suffuse/util.go

```diff
@@ -97,3 +97,9 @@ func ScratchFile() Path {
   deleteOnExit = append(deleteOnExit, path)
   return path
 }
+
+// https://github.com/golang/go/issues/1312
+func FileExists(path string) bool {
+  _, err := os.Stat(path)
```

Comment:
Haha, yeah.

All good points. My knowledge of the domain (files) is still very limited, so it's good you alert me to the `Lstat` one. I agree on the separation of the two. I hate boolean (or similar) parameters as it. Another idea would be something like:

```
y := x.FollowingLinks()
y.Exists()
```

In any case, I will definitely move it to `Path`.